assets/airports/eidw.json

```diff
@@ -13,137 +13,330 @@
+                ["N53.402043200", "W5.458241200"],
+                ["N53.386841700", "W5.459079200"],
+                ["N53.371686700", "W5.461242600"],
+                ["N53.356619100", "W5.464724400"],
+                ["N53.341679500", "W5.469514000"],
+                ["N53.326908000", "W5.475597500"],
+                ["N53.312344600", "W5.482957600"],
+                ["N53.298028600", "W5.491573800"],
+                ["N53.283998700", "W5.501422200"],
+                ["N52.333073347", "W5.500648488"],
+                ["N52.614555848", "W6.618507374"],
+                ["N53.090560313", "W7.333305348"],
+                ["N53.752608525", "W7.333991994"]
             ]
         }
     ],
     "fixes": {
-        "_EIDW243060": ["N52d53.92m0", "W07d41.12m0"],
-        "_EIDW064060": ["N53d54.70m0", "W04d48.33m0"],
-        "_EIDW034060": ["N54d16.99m0", "W05d25.02m0"],
-        "_EIDW305060": ["N53d55.61m0", "W07d43.18m0"],
-        "OLAPO": ["N53d46m49.00", "W7d17m41.00"],
-        "OSGAR": ["N53d02m57.90", "W7d16m12.80"],
+        "_BAL1":     ["N53.516188800", "W6.319345800"],
+        "_BAL10":    ["N53.535910700", "W6.364793300"],
+        "_BAL11":    ["N53.536316400", "W6.371251600"],
+        "_BAL12":    ["N53.536316400", "W6.377746100"],
+        "_BAL13":    ["N53.535910700", "W6.384204400"],
+        "_BAL14":    ["N53.535104000", "W6.390554500"],
+        "_BAL15":    ["N53.533905400", "W6.396726200"],
+        "_BAL16":    ["N53.532328600", "W6.402650800"],
+        "_BAL17":    ["N53.530391600", "W6.408263100"],
+        "_BAL18":    ["N53.528116100", "W6.413501400"],
+        "_BAL19":    ["N53.525527700", "W6.418308800"],
+        "_BAL2":     ["N53.519530900", "W6.322570100"],
+        "_BAL20":    ["N53.522655300", "W6.422633100"],
+        "_BAL21":    ["N53.519530900", "W6.426427600"],
+        "_BAL22":    ["N53.516188800", "W6.429651900"],
+        "_BAL23":    ["N53.512665700", "W6.432271700"],
+        "_BAL24":    ["N53.509000100", "W6.434259500"],
+        "_BAL25":    ["N53.505231800", "W6.435594300"],
+        "_BAL26":    ["N53.501401700", "W6.436262400"],
+        "_BAL3":     ["N53.522655400", "W6.326364600"],
+        "_BAL4":     ["N53.525527800", "W6.330688900"],
+        "_BAL5":     ["N53.528116100", "W6.335496200"],
+        "_BAL6":     ["N53.530391600", "W6.340734600"],
+        "_BAL7":     ["N53.532328600", "W6.346346900"],
+        "_BAL8":     ["N53.533905400", "W6.352271500"],
+        "_BAL9":     ["N53.535104000", "W6.358443100"],
+        "_BATED1":   ["N53.342173000", "W6.175542100"],
+        "_BATED10":  ["N53.321350200", "W6.115311100"],
+        "_BATED11":  ["N53.321350200", "W6.107283400"],
+        "_BATED12":  ["N53.321853300", "W6.099300000"],
+        "_BATED13":  ["N53.322853700", "W6.091449500"],
+        "_BATED14":  ["N53.324340000", "W6.083818700"],
+        "_BATED15":  ["N53.326295500", "W6.076491600"],
+        "_BATED16":  ["N53.328698100", "W6.069548600"],
```

Comment:
Are these used for DME arcs? If so, they should be renamed in fix-radial-distance form.